app/src/main/java/cash/bchd/android_neutrino/MainActivity.java

```diff
@@ -369,9 +368,11 @@ private void copyToClipboard(String data) {
         Object clipboardService = getSystemService(CLIPBOARD_SERVICE);
         final ClipboardManager clipboardManager = (ClipboardManager) clipboardService;
         ClipData clipData = ClipData.newPlainText("Source Text", data);
-        clipboardManager.setPrimaryClip(clipData);
-        Snackbar snackbar = Snackbar.make(mCLayout, "Address copied clipboard.", Snackbar.LENGTH_LONG);
-        snackbar.show();
+        if (clipboardManager != null) {
+            clipboardManager.setPrimaryClip(clipData);
+            Snackbar snackbar = Snackbar.make(mCLayout, "Address copied clipboard.", Snackbar.LENGTH_LONG);
```

Comment:
Shouldn't this string be in the XML below?

Reply:
yes